ethcore/src/state.rs

```diff
@@ -177,7 +179,7 @@ impl State {
 
 	/// Mutate storage of account `address` so that it is `value` for `key`.
 	pub fn storage_at(&self, address: &Address, key: &H256) -> H256 {
-		self.get(address, false).as_ref().map_or(H256::new(), |a|a.storage_at(&AccountDB::new(self.db.as_hashdb(), address), key))
+		self.get(address, false).as_ref().map_or(H256::new(), |a| a.storage_at(&AccountDB::new(self.db.as_hashdb(), address), key))
```

Comment:
Citing @arkpar "Smart contracts don't use RLP. There's nearly 0% chance you'll get an RLP in the contract storage"